source/jormungandr/tests/ptref_tests.py

```diff
@@ -221,6 +245,8 @@ def test_route(self):
         assert com[0]['type'] == 'standard'
         assert com[0]['value'] == "I'm a happy comment"
 
+        self._test_links(response, 'routes')
```

Comment:
You can still take care of my general remark on links consistency in a dedicated PR  :wink: